templates/search_page.html

```diff
@@ -140,9 +142,31 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
 
                 </div>
 
+                <div class="ps-1 pe-0 mt-1" id="priceDiv">
+                    <select name="price" class="form-select border border-secondary"
+                        aria-label=".form-select-sm example" id="priceToSelect" onchange="getRandomInt(this)">
+                        <option>price</option>
+                        <option value="0">$</option>
+                        <option value="1">$$</option>
+                        <option value="2">$$$</option>
+                        <option value="3">$$$$</option>
+                        <option value="4">Max</option>
+                        <option value="0,1,2,3,4">Surprise</option>
+                        <script>
+                            function getRandomInt() {
```

Comment:
Can we rename this to randomPriceRange? Try avoid using get in names unless it's http handling method with Get